krcal/core/map_functions.py

```diff
@@ -203,7 +203,7 @@ def amap_max(amap : ASectorMap)->FitMapValue:
                       mapinfo   = amap.mapinfo)
 
 
-def amap_min(amap : ASectorMap)->FitMapValue:
+def amap_min(amap : ASectorMap)->ASectorMap:
```

Comment:
I think here it should indeed be FitMapValue since we are storing only one float per map?